File: C/Program26.c
//////////////////////////////////////////////////////////////////////////
//                                                                      //
//  File name :     Program26.c                                         //
//  Description :   Used to display sum of digits from given number     //
//  Author :        Pallavi Omprakash Malewar                           //
//  Date :          27/06/2025                                          //
//                                                                      //
//////////////////////////////////////////////////////////////////////////


#include<stdio.h>

int SumOfDigits(int iNumber)
{
    int iSum = 0 ;
    int iDigit = 0 ;

    while( iNumber != 0 )
    {
        iDigit = iNumber % 10 ;
        iNumber = iNumber / 10 ;
        iSum = iSum + iDigit ;
    }

    return iSum;
}


int main()
{
    int iValue = 0 ;
    int iRet = 0 ;

    printf("Enter Number : ") ;
    scanf("%d" , &iValue) ;

    iRet = SumOfDigits(iValue) ;

    printf("Summation of digits are : %d\n" , iRet) ;

    return 0 ;

}

File: JAVA/Program26.java
//////////////////////////////////////////////////////////////////////////////////////////
//                                                                                      //
//  File name :     Program26.java                                                      //
//  Description :   Used to Print Given Pattern                                         //
//  Author :        Pallavi Omprakash Malewar                                           //
//  Date :          25/08/2025                                                          //
//                                                                                      //
//////////////////////////////////////////////////////////////////////////////////////////


// Input : 4
// Output : -4  -3  -2  -1  0   1   2   3   4



import java.util.Scanner;

class Pattern
{
    public void Display(int iNo)
    {
        int iCount = 0;

        if(iNo < 0)
        {
            iNo = -iNo;     // Filter
        }

        for(iCount = -iNo ; iCount <= iNo ; iCount++)       // O(2N)
        {
            System.out.print(iCount + "\t");
        }
        System.out.println("");
    }
}

class Program26
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.print("Enter Number : ");
        iValue = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue);

    }
}

// Time Complexity O(N)
// N is a Natural Number
// Where N >= 0 & N <= Infinity